src/app/lightning/get-balances.ts

```diff
@@ -23,6 +23,21 @@ export const getTotalBalance = async (): Promise<Satoshis | ApplicationError> =>
   return sumBalances(balances)
 }
 
+export const getInboundBalance = async (): Promise<Satoshis | ApplicationError> =>
```

Comment:
I don't see a reason to cache the value here?

Reply:
addressed in [1103899](https://github.com/GaloyMoney/galoy/pull/2057/commits/11038996b32a3643fb143ee24a9dbec792cb7a1a)